logging/logging.go

```diff
@@ -18,6 +20,10 @@ var (
 	// GlobalLogLevel should be used whenever a zap logger is created that wants to obey the debug
 	// flag from the CLI or robot config.
 	GlobalLogLevel = zap.NewAtomicLevelAt(zap.InfoLevel)
+
+	// DisableLogDeduplication controls whether to disable de-duplicating noisy
+	// logs; defaults to false and can be specified in robot config.
+	DisableLogDeduplication = atomic.Bool{}
```

Comment:
I get why we use this name for the config flag (we want this to be on for robots by default), but I'm curious if you gave any consideration and/or had a reason why this global shouldn't be `EnableLogDeduplication`/`DeduplicateLogs`? The two benefits I see:
* Easier to read symbols in predicates that are in the affirmative
* The "least surprising behavior" (to a programmer) is for deduplication to be _off_. I think it's going to be more common to add tests that do not want deduplication. And moreover forget* it's happening. And tests that do need deduplication will know to turn it on.